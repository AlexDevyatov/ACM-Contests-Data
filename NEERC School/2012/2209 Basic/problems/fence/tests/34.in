11
-85 -51
-82 -79
-63 -82
-34 -83
5 -82
62 -72
84 -59
74 74
39 83
-50 85
-85 60
-3 37
