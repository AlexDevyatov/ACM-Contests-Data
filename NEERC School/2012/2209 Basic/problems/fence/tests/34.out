12
-85 -51
-82 -79
-63 -82
-34 -83
5 -82
62 -72
84 -59
74 74
-3 37
39 83
-50 85
-85 60
